Django-Start-Project.md

```diff
@@ -0,0 +1,265 @@
+# Creating a Project with Django
+
```

Comment:
For now, link [Previous](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Install-Django-Flask) as I have.